chapters/lang/expressions.tex

```diff
@@ -189,21 +191,77 @@ \subsection{Term comparison operators}
 Number (Int or Float) < Char < String < Regex
 \end{lstlisting}
 
-Integers are comparable with integers in usual way. Floating-point numbers are comparable with floating-point numbers according to IEEE-754\cite{IEEE754} rules. For \lstinline{===} and \lstinline{!==} operators comparing integer to floating-point number (and vice-versa) always fails, otherwise integer is converted to float before comparison.
+Integers are comparable with integers in the usual way. Floating-point numbers are comparable with floating-point numbers according to IEEE-754\cite{IEEE754} rules. For \lstinline{===} and \lstinline{!==} operators comparing integer to floating-point number (and vice-versa) always fails, otherwise integer is converted to float before comparison.
 
 Characters are compared using their byte representation. Strings and regular expressions are compared character by character. For \lstinline{===} and \lstinline{!==} operators comparisons between characters, strings and regular expressions always fail, otherwise characters and regular expressions are always converted to strings before \lstinline{==} and \lstinline{!=} comparison.
 
-TODO(Ania): which term is returned?
+No matter what comparison was made, its result consists of term copied from the second argument.
```

Comment:
Are we sure it is copied? I think not always, reference will be passed in most situations. I'd change this to *result term is equal to second argument's term* just to be safe